exampleSite/content/index/contact.md

```diff
@@ -12,7 +12,7 @@ title = "Contact fragment"
 subtitle  = "*not working on demo page*"
 
 # PostURL can be used with backends such as mailout from caddy
-postURL = "https://example.com/mailout" #default: formspree.io
+posturl = "https://example.com/mailout" #default: formspree.io
```

Comment:
So you decided against user facing camelCase?

Reply:
No. We agreed on changing to camelCase on the front matter. I'll make the changes.